atc/db/resource_factory.go

```diff
@@ -40,12 +43,28 @@ func (r *resourceFactory) VisibleResources(teamNames []string) ([]Resource, erro
 		return nil, err
 	}
 
+	return r.parseResources(rows)
+}
+
+func (r *resourceFactory) AllResources() ([]Resource, error) {
+	rows, err := resourcesQuery.
+		OrderBy("r.id ASC").
+		RunWith(r.conn).
+		Query()
+	if err != nil {
+		return nil, err
+	}
+
+	return r.parseResources(rows)
+}
+
+func (r *resourceFactory) parseResources(resourceRows *sql.Rows) ([]Resource, error) {
```

Comment:
Typically we would call this `scanResources` and it would be a package level function (instead of being tied to the resourceFactory). That way it can be reused in other places if needed. See `scanResource` as an example.